scipy/stats/morestats.py

```diff
@@ -182,9 +182,11 @@ def kstat(data, n=2):
     See Also
     --------
     kstatvar: Returns an unbiased estimator of the variance of the k-statistic.
+    moment: Returns the n-th central moment about the mean for a sample.
 
     Notes
     -----
+    The input data gets raveled prior to calculating the cumulants.
```

Comment:
I'd move this to the parameter description for ``data``. Something like ``Input data.  Note that n-D input gets flattened.``